prefs/prefs.go

```diff
@@ -0,0 +1,56 @@
+package prefs
+
+import (
+	"pkg.re/essentialkaos/ek.v8/knf"
+	"pkg.re/essentialkaos/ek.v8/fmtc"
+	"os"
+	"pkg.re/essentialkaos/ek.v8/fsutil"
+)
+
+// ////////////////////////////////////////////////////////////////////////////////// //
+
+const (
+	STORAGE_URL  = "storage:url"
+	STORAGE_PATH = "storage:path"
+)
+
+// ////////////////////////////////////////////////////////////////////////////////// //
+
+type Preferences struct {
+	StorageURL string
```

Comment:
Thanks. I added `make fmt` and `gomakegen` commands to the pre-commit hook. I suppose now there is no chance to forgot to format and generate Makefile properly.